pyccl/correlation.py

```diff
@@ -19,20 +24,31 @@
     'l-': const.CCL_CORR_LM,
 }
 
+
 def correlation(cosmo, ell, C_ell, theta, corr_type='gg', method='fftlog'):
     """Compute the angular correlation function.
 
     Args:
         cosmo (:obj:`Cosmology`): A Cosmology object.
-        ell (array_like): Multipoles corresponding to the input angular power spectrum.
+        ell (array_like): Multipoles corresponding to the input angular power
+                          spectrum.
         C_ell (array_like): Input angular power spectrum.
-        theta (float or array_like): Angular separation(s) at which to calculate the angular correlation function (in degrees).
-        corr_type (string): Type of correlation function. Choices: 'GG' (galaxy-galaxy), 'GL' (galaxy-shear), 'L+' (shear-shear, xi+), 'L-' (shear-shear, xi-).
-        method (string, optional): Method to compute the correlation function. Choices: 'Bessel' (direct integration over Bessel function), 'FFTLog' (fast integration with FFTLog), 'Legendre' (brute-force sum over Legendre polynomials).
-    
-    Returns:
-        float or array_like: Value(s) of the correlation function at the input angular separations.
+        theta (float or array_like): Angular separation(s) at which to
+                                     calculate the angular correlation
+                                     function (in degrees).
+        corr_type (string): Type of correlation function. Choices:
+                            'GG' (galaxy-galaxy), 'GL' (galaxy-shear),
+                            'L+' (shear-shear, xi+),
+                            'L-' (shear-shear, xi-).
+        method (string, optional): Method to compute the correlation function.
+                                   Choices: 'Bessel' (direct integration over
+                                   Bessel function), 'FFTLog' (fast
+                                   integration with FFTLog), 'Legendre' (
+                                   brute-force sum over Legendre polynomials).
```

Comment:
This would read better with one choice per line starting with the method name at the front, rather than wrapping.
Likewise for corr_type, it would read better to have GG and GL each on their own line.

Reply:
Yes makes sense. I am going to merge and move this into an issue to be addressed later with other doc fixes.